frame/balances/src/lib.rs

```diff
@@ -192,6 +192,20 @@ pub mod pallet {
 	#[pallet::config]
 	pub trait Config<I: 'static = ()>: frame_system::Config {
 		/// The balance of an account.
+		///
+		/// It is important to note that this type is all about the minimum presentation of
+		/// your token, just like a Satoshi, which is the smallest Bitcoin representation.
+		///
+		/// When dealing with internal arithmetic operations, it is also worth tracking the same
+		/// decimals defined in the node chain specification (`tokenDecimals`) for proper number
+		/// evaluation and user UI display. Take for example an hypothetical case where
+		/// `type Balance = u128;` and `tokeDecimals` is `18`.
```

Comment:
I actually do not agree with this. When dealing with internal arithmetic operations, you should always work with the lowest unit, and never do manipulations to try and take into account `BASE`

If you need some pallet which takes some `BASE` (which i think to start is not a good idea), it should be brought into scope with some runtime configuration, rather than programmed like it is below.

A pallet should not need to know what 1 DOT is or anything like that. Only what configurable variables like `MinimumBalance` should be.

Reply:
No problem, I will re-elaborate these statements.
The idea here is give better documentation so thanks for the feedback.